[DOCS] Updates what's new in 8.4 (#138847)
elastic/kibana: docs/user/whats-new.asciidoc

coming::[8.4.0]

[discrete]
[[highlights-8.4-release-docs]]
=== Elastic release docs now in one place

> Want to learn about all the latest Elastic features and updates?

Reviewer: ```suggestion
With the new https://www.elastic.co/guide/en/welcome-to-elastic/current/new.html[Check out the latest from Elastic] page, you can now easily find all the latest information about Elastic features and updates. From release highlights, to breaking changes, to bug fixes, and more, you can now easily access all of this information in a single location.
```

> We’ve made it easy for you to find everything you need to know with

Reviewer: ```suggestion
```

> the new

Reviewer: ```suggestion
```

> https://www.elastic.co/guide/en/welcome-to-elastic/current/new.html[Check out the latest from Elastic] page.

Reviewer: ```suggestion
```

> Now you can access all release information you need in a single location.

Reviewer: ```suggestion
```

[role="screenshot"]

> image:images/highlights-release-docs.png[New release docs page on elastic.co, width=75%]

Reviewer: This image looks like the new release docs page includes only Observability. 

This is a better image:
<img width="560" alt="Screen Shot 2022-08-15 at 11 20 49 AM" src="https://user-images.githubusercontent.com/38990715/185459152-a6c32569-3a85-406d-a36c-5e57d3a9d8d0.png">
